content/configuration/index.md

```diff
@@ -69,6 +71,25 @@ password = ""           # postgis database password
 srid = 3857             # The default srid for this provider. If not provided it will be WebMercator (3857)
 ```
 
+### GeoPackage
```

Comment:
Please make a note that this data provider is only available in the _cgo version of the app. Or if they compile it them selfs and disable cgo, it will not be available.